packages/aft/lib/src/commands/link_command.dart

```diff
@@ -59,13 +59,29 @@ Map<String, String> _collectDependencies(
   return dependencyPaths;
 }
 
+/// Removes the `pubspec_overrides.yaml` file for [package].
+Future<void> _removePubspecOverride(PackageInfo package) async {
+  final pubspecOverrides = File(
+    path.join(package.path, 'pubspec_overrides.yaml'),
+  );
+  if (await pubspecOverrides.exists()) {
+    await pubspecOverrides.delete();
+  }
+}
+
 /// Creates a `pubspec_overrides.yaml` file for [package].
 Future<void> _createPubspecOverride(
   PackageInfo package,
   Map<String, String> dependencyPaths,
   // To be merged in
   Map<String, Dependency> existingDependencyOverrides,
 ) async {
+  // Unlike every other package, aft must work without pubspec_overrides.
+  // It's counter productive to generate it and its presence can interfere
+  // with local development.
+  if (package.name == 'aft') {
+    await _removePubspecOverride(package);
```

Comment:
Could we prevent the override file from being created in the first place?

Reply:
Whoops. Should've been a `return`. Fixed 👍